docs/manual/backend/backend-keyboard-shortcuts.en.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Keyboard shortcuts"
+description: "To speed up the workflow when working with Contao, there are several keyboard shortcuts in the backend, 
+with which certain commands can be called directly."
+url: "administration-area/back-end-keyboard-shortcuts"
+weight: 20
+---
+
+To speed up the workflow when working with Contao, there are several keyboard shortcuts in the backend that allow you to
+call up certain commands directly. For example, the `Save` button is always located at the end of an input form, so that 
+it can only be clicked with the mouse when the page is completely scrolled down.
+
+Normally this is not a problem, because when filling out a form you work from top to bottom. But 
+if you later want to change only a single value at the beginning of the form, the Button is not accessible without scrolling. 
+In this case, the experienced Contao user will be able to access the Keyboard shortcut `[Alt]+[S]` even without scrolling 
```

Comment:
To keep it more simple and include even the "non-experienced Contao users" (😉) I would simply write.

```suggestion
In this case, you are able to access the Keyboard shortcut `[Alt]+[S]` even without scrolling 
```